demo/deploy-bookstore.sh

```diff
@@ -23,6 +23,8 @@ spec:
   ports:
   - port: 80
     name: bookstore-port
+  selector:
```

Comment:
I have added the selector on apex service back @draychev, @shashankram

Reply:
FYI, this isn't needed but putting it here doesn't cause any harm for this demo.

Reply:
yes, I added it back since we had it previously